frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -126,6 +128,7 @@ const TitleText = styled.p`
   font-weight: 700;
```

Comment:
We may want to make the font size for the titles  in terms of vh/vw

Reply:
I think the titles are good to stay at the full size, since they are like an important part if that makes sense.

Reply:
That's good with me!